feature/gribi/otg_tests/static_lsp/static_lsp_test.go

```diff
@@ -0,0 +1,367 @@
+package mpls_static_label
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/otgutils"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/otg"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	ipv4PrefixLen = 30
+	ipv6PrefixLen = 126
+	mplsLabel1    = uint32(100001)
+	mplsLabel2    = uint32(100002)
```

Comment:
Labels do not match the readme (100,001 instead of 1,000,001)

Also with consts in golang uint32 conversion is redundant

Reply:
Addressed this in latest commit